lerobot/configs/default.yaml

```diff
@@ -5,10 +5,20 @@ defaults:
 
 hydra:
   run:
+    # Set `dir` to where you would like to save all of the run outputs. If you run another training session
+    # with the same value for `dir` its contents will be overwritten unless you set `resume` to true.
     dir: outputs/train/${now:%Y-%m-%d}/${now:%H-%M-%S}_${env.name}_${policy.name}_${hydra.job.name}
   job:
     name: default
 
+# Set `resume` to true to resume a previous run. In order for this to work, you will need to make sure
+# `hydra.run.dir` is the directory of an existing run with at least one checkpoint in it.
+# Note that when resuming a run, the default behavior is to use the configuration from the checkpoint,
+# regardless of what's provided with the training command at the time of resumption.
+resume: false
+# Set `override_config_on_resume` to true to use the provided configuration instead of the one from the
+# checkpoint.
+override_config_on_resume: false
```

Comment:
Could you add a bit more info? I understood `override_config_on_resume`, but now I forgot how it is used. I know the info is somwhere in 5_resume_training but it would be ideal to have it here as well.